crates/typst-library/src/html/dom.rs

```diff
@@ -522,6 +522,11 @@ pub mod tag {
                 | self::p
                 | self::pre
                 | self::search
+                | self::table
```

Comment:
Since these are not actually `display: block`, this seems incorrect. We should rather add these to `is_pretty` in `typst-html`.